coverage/__init__.py

```diff
@@ -10,13 +10,13 @@
 
 import sys
 
-from coverage.version import __version__, __url__, version_info
+from coverage.version import __version__ as __version__, __url__ as __url__, version_info as version_info
```

Comment:
Why add these `as` clauses? They're the same names they already were.

Reply:
In python typing this nonsense is semantically understood as "reexport this import as a public member of the module".

I believe this crude hack was made because they didn't wan't to introduce any new syntax for typing.

It is mentioned in [pep-484](https://peps.python.org/pep-0484/#stub-files)